kit/motor_board.md

```diff
@@ -22,12 +22,13 @@ Board Diagram
 Indicators
 ----------
 
-| LED                    | Meaning                 | Initial power-up state
-|------------------------|-------------------------|------------------------------
-| Power                  | The board is powered    | On
-| M{0,1} Speed/Direction | Brightness indicates speed, colour indicates direction | Off
-| USB Power              | The USB interface is powered | On
-| USB Data               | Data is being transferred to/from the board | Off
+| LED                    | Colour     | Meaning                                                | Initial power-up state |
+|------------------------|------------|--------------------------------------------------------|------------------------|
+| Power                  | Green      | The board is powered                                   | On                     |
+| Power                  | Red        | If lit, the polarity is wrong for Power In             | Off                    |
```

Comment:
To match the pattern of the below line, we should drop these to 1 line.

Reply:
The other LEDs do not have multiple meaningful colours.

Reply:
There's an existing precedent for the multi-line approach in the [power board docs](https://studentrobotics.org/docs/kit/power_board#indicators), it would be great if we could use that.

Reply:
> The other LEDs do not have multiple meaningful colours.

The line below my comment does exactly this. We should be consistent within the same table